src/crypto.ts

```diff
@@ -0,0 +1,5 @@
+export function hash(data: string, salt: string): string {
+  const unhashed_string = data + salt;
+  const hashed_string: string = require('crypto').createHash('sha256').update(unhashed_string).digest('base64');
+  return hashed_string;
+}
```

Comment:
```suggestion
import * as crypto from 'crypto';

export function hash(data: string, salt: string): string {
  const unhashed_string = data + salt;
  const hashed_string: string = crypto.createHash('sha256').update(unhashed_string).digest('base64');
  return hashed_string;
}
```

Assim mantemos os imports do Javascript "moderno".